AirCasting/SessionViews/SessionCellView.swift

```diff
@@ -27,7 +28,10 @@ struct SessionCellView: View {
             thresholds: Array(thresholds))
             if !isCollapsed {
                 VStack(alignment: .trailing, spacing: 40) {
-                    pollutionChart
+                    #warning("The stream should be chosen based on users selection")
+                    if let streams = session.measurementStreams {
+                        pollutionChart(stream: streams[0] as! MeasurementStreamEntity)
```

Comment:
There are cases where the stream is not created so assuming the array has elements here is not safe